custom_components/tuya_local/devices/thermoval-aquati-waterheater.yaml

```diff
@@ -0,0 +1,92 @@
+name: Thermoval Aqua Ti Water Heater
```

Comment:
The top level name should be without branding, as other products may match the config. Branding belongs under products, where it will be used in future if the product id matches.